main.py

```diff
@@ -10,8 +10,8 @@
 BOT_NAME = os.getenv('DISCORD_BOT_NAME')
 
 intents = discord.Intents.all()
-bot = commands.Bot(command_prefix='!', intents=intents)
 
+bot = commands.Bot(command_prefix='/', intents=intents)
```

Comment:
```suggestion
bot = commands.Bot(command_prefix='!', intents=intents)
```

Change back to '!' or a different symbol, having a '/' is confusing with the slash command menu since it's a different type of command.